documentation/book/assembly-downgrading-kafka-versions.adoc

```diff
@@ -0,0 +1,26 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+
+[id='assembly-downgrading-kafka-versions-{context}']
+
+= Kafka downgrade
+
+Kafka version downgrades are performed using the Cluster Operator.
+
+Whether and how the Cluster Operator performs a downgrade depends on the differences between versions of:
+
+* Interbroker protocol
+* Log message format
+* Zookeeper
+
+If the target downgrade version of Kafka has the same log message format version as the current verison, the Cluster Operator downgrades by performing a single rolling restart of the brokers.
+
+If the target downgrade version of Kafka has a different log message format version, downgrading is only possible if the running cluster has _always_ had `log.message.format.version` set to the version used by the downgraded version.
```

Comment:
I think it would be good to make this more visible.

Reply:
I have split out to a new section and used bullets to make it more clear.